lib/evmone/execution.cpp

```diff
@@ -1518,6 +1525,7 @@ exec_fn_table create_op_table_petersburg() noexcept
 exec_fn_table create_op_table_istanbul() noexcept
 {
     auto table = create_op_table_petersburg();
+    table[OP_BALANCE] = op_selfbalance;
```

Comment:
Need to use the correct opcode.

Reply:
Just use `0x46`.

Reply:
Well, the bigger work left for 1884 is repricing the opcodes and that requires a change in EVMC, because the instruction metrics live there, if I'm not wrong?